helpers/posts.rb

```diff
@@ -27,4 +27,25 @@ def capitalize(tagname)
     tagname = tagname.slice(0, 1).capitalize + tagname.slice(1..-1)
     tagname.gsub(/-[a-z]/, &:upcase)
   end
+
+  # Define which partial should be used
+  def article_preview
```

Comment:
```suggestion
  def article_preview_partial_path
```

would describe the function better :)

Reply:
Done !